### DISCUSSION TITLE
Expedition 3 Redux

### DISCUSSION THREAD

**Discussion by bedev0001** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#35] [General]
In Phase 3, you are required to repair your ship using Crystal Sulphide, which was originally found in underwater areas on Gisto Major. However, the planet has now transformed into a hot, arid planet with no water. Additionally, during this expedition, you're unable to use the teleporter or purchase a new ship.  That means, in this expedition you get beached due to the galactic planetary biom changes that have accured. I looked up if other people had this issue, but the solutions I found did not worked out for me.

I'm not sure if there is any solution to this problem, given the circumstances, because we can not play these missions online to get helped out.

**Comment by cwmonkey** [comment]
@bedev0001 I think I got mine from buried minerals, you can see them on your scanner, similar to buried technology modules.

**Reply by bedev0001** [reply]
You sure? I only got Glowing Mineral from buried minerals and the nms wiki does not mantion this. I will give it a try tough.

**Reply by bedev0001** [reply]
Update: this works! I just realized now, that the glowing minerals are like the geodes. The chances to get Crystal Sulphide from it are very low tough, so I recommend to use the duplication bug for quick progress.

**Comment by mozai** [comment]
I just started Expedition 3 (not 3bis) and I'm dropped on a planet called Coul Major (High-Temperature Planet orbiting Udhlek, 04AD:0082:0E32:0033 or boat/sunset/diplo/diplo/sunset/diplo/boat/face/eclipse/tent/voxel/atlas).  I used the trick above, finding underground mineral structures, diffing up glowing crystals, and cracking them hoping to find 3 crystal sulphide.

After digging up a couple dozen to get three, next I took a long walk a 950u tall mountain to climb.  About fifteen minutes's west of where my ship is stuck, I found a huge body of water with plenty of crystal sulphide.  bleah.

Seems I'm out of luck for the mountain achievement, since the land seems to truncate at 450u -- hills that rise up to that height and then level off into flat plataeaus.  I can't even matter-manipulate above that height.  Hopefully I can get this achievement/requirement on another planet after I lift off from this one to head for the first waypoint.

**Reply by bedev0001** [reply]
Yeah thank you, these crystals were a pain for me too. I just got some glowing crystals and used the dublication bug, spared me some time.

About the mountain achievement there is a trick, if your jetpack is completly upgraded, you can look for a planet or moon with low gravity and just press the spacebar to get it. So you need to look, that your jetpack recovers fast enough to push you constantly upwards. That is how I did it.

**Reply by bedev0001** [reply]
Also you can just build a base upward to the sky, like the people do, when they try to built a custom orbit space station.
That should do it too.

**Reply by cwmonkey** [reply]
Ah, so it's the same as it was a while ago. Back then I was driving to... something, I can't remember what, but also hit a body of water after like 20-30 minutes. I think by that time I already had my crystal sulfide, though.
